video/image_writer.c

```diff
@@ -96,7 +103,12 @@ static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image, FILE *fp
 
     av_init_packet(&pkt);
 
-    struct AVCodec *codec = avcodec_find_encoder(ctx->opts->format);
+    struct AVCodec *codec;
+    if (ctx->opts->format == AV_CODEC_ID_WEBP)
+        codec = avcodec_find_encoder_by_name("libwebp"); // non-animated encoder
```

Comment:
So ffmpeg returns the wrong thing here? Are there any disadvantages? This looks really wrong.